Source/WebCore/page/scrolling/ScrollingTreeScrollingNode.cpp

```diff
@@ -160,6 +160,11 @@ bool ScrollingTreeScrollingNode::canHandleWheelEvent(const PlatformWheelEvent& w
     return eventCanScrollContents(wheelEvent);
 }
 
+bool ScrollingTreeScrollingNode::allowsScrolling() const
+{
+    return m_scrollableAreaParameters.allowsHorizontalScrolling || m_scrollableAreaParameters.allowsVerticalScrolling;
+}
+
```

Comment:
`check-webkit-styles` told me not to inline it because of the class-wide `WEBCORE_EXPORT` but the class is already full on inline methods :)